pkg/ingester/user_tsdb.go

```diff
@@ -324,6 +333,52 @@ func (u *userTSDB) PostDeletion(metrics ...labels.Labels) {
 	}
 }
 
+func (u *userTSDB) ephemeralSeriesCallback() tsdb.SeriesLifecycleCallback {
+	return seriesLifecycleCallback{
+		preCreation:  u.ephemeralPreCreation,
+		postCreation: u.ephemeralPostCreation,
+		postDeletion: u.ephemeralPostDeletion,
+	}
+}
+
+// PreCreation implements SeriesLifecycleCallback interface.
```

Comment:
[nit] Rename these comments to use the new funciton name.

Reply:
I can either write something like `// persistentPreCreation is used as pre-creation callback from tsdb.SeriesLifecycleCallback interface for persistent storage.` or just remove the comment, as the usage is clear from

```golang
func (u *userTSDB) persistentSeriesCallback() tsdb.SeriesLifecycleCallback {
	return seriesLifecycleCallback{
		preCreation:  u.persistentPreCreation,
		postCreation: u.persistentPostCreation,
		postDeletion: u.persistentPostDeletion,
	}
}
```

method. I prefer removing the obsolete comment.

Reply:
I also agree the comment is not much useful.

Reply:
Removed obsolete comments here: https://github.com/grafana/mimir/pull/4004/commits/5eab1b64781f69e9301d3a0367588e317c6fa559